components/Table/Table.js

```diff
@@ -279,10 +277,12 @@ export class Table {
   render(data, totalAmount) {
     this.clearTable();
     this.renderPagesAmount(totalAmount);
+    this.copyOfData = data;
```

Comment:
Copy of the which data is it?
We're passing data to the method as an argument but we do not have this data in the class to say, that this is the copy